providers/dnsimple/auditrecords.go

```diff
@@ -11,7 +11,7 @@ import (
 func AuditRecords(records []*models.RecordConfig) []error {
 	a := rejectif.Auditor{}
 
-	a.Add("MX", rejectif.MxNull) // Last verified 2023-03
+	// DNSimple sent email 2024-05-16 announcing that they now support Null MX records.
```

Comment:
If you ask me, I would personally leave this comment out. And just delete the original line. 

```suggestion
```

Reply:
I think that when things change, and people are swearing they thought X was the case, a comment providing a clue helps.  Changed in observed behavior, and when, are ideal use-cases for comments explaining both what is present and what is deliberately not present.

Reply:
Delete the line and trust that the PR description is documentation enough. 

If we were taking away a feature, more documentation would be warranted. However I've never seen anyone complain about losing restrictions.